api/webendpoints_test.go

```
@@ -116,9 +116,10 @@ func hasRoute(r *mux.Router, path, method string) bool {
 }
 
 func createRequestWithAuth(method, url string, body io.Reader) (*http.Request, error) {
```

Does this function need to return an error if it never will?